base/applications/rapps/include/rosui.h

```diff
@@ -1,15 +1,19 @@
 /*
  * PROJECT:     ReactOS UI Layout Engine
  * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
- * FILE:        base/applications/rapps/include/rosui.h
  * PURPOSE:     ATL Layout engine for RAPPS
- * COPYRIGHT:   Copyright 2015 David Quintana           (gigaherz@gmail.com)
+ * COPYRIGHT:   Copyright 2015 David Quintana (gigaherz@gmail.com)
  */
+
 #pragma once
 
 #include <atlwin.h>
 
-template<class T, INT GrowthRate = 10>
+//
+// "Basic" types / container(s)
+//
+
+template</*class TDeriv,*/ typename T, INT GrowthRate = 10>
```

Comment:
```suggestion
template<typename T, INT GrowthRate = 10>
```

Reply:
Well, this was about the question I had about this class, with regards to the `CUiContainer` (see large comment down below)